recipe/meta.yaml

```diff
@@ -1,32 +1,42 @@
-{% set name = "wxPython" %}
-{% set version = "4.0.0a2" %}
-{% set sha256 = "3acd5160d69f56b588476d1907eff67b53ad58fb2eba4dedabcdc3490d170338" %}
```

Comment:
This is more readable IMO and I have a script that updates the recipe automatically when the only jinja var is the version number.